WcaOnRails/app/javascript/edit-events/EditEvents.jsx

```diff
@@ -17,7 +17,7 @@ export default class EditEvents extends React.Component {
     let onSuccess = () => this.setState({ savedWcifEvents: _.cloneDeep(wcifEvents), saving: false });
     let onFailure = () => this.setState({ saving: false });
 
-    saveWcifPart(competitionId, '/events', wcifEvents, onSuccess, onFailure);
+    saveWcifPart(competitionId, 'events', wcifEvents, onSuccess, onFailure);
```

Comment:
How do you feel about renaming `saveWcifPart` to `saveWcif` and combining the second and third arguments, so this call would look this this instead:

```javascript
saveWcif(competitionId, { events: wcifEvents }, onSuccess, onFailure);
```

Reply:
I like that! Will update it that way =)